platform/ui/src/views/StudyList/components/Header.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withTranslation } from 'react-i18next';
 
-import { IconButton, Icon } from '@ohif/ui';
+import { IconButton, Icon, NavBar } from '@ohif/ui';
```

Comment:
Can we demo the Header component in a category? I want to be able to see it in something other than the generic "Components" category.